Ugh.
```
Command '['/entrypoint.sh', 'build']' in image 'metalk8s-ui-builder:latest' returned non-zero exit status 1: b'\n> metalk8s-platform-ui@0.1.0 build /home/node\n> react-app-rewired --max_old_space_size=3072 build\n\nCreating an optimized production build...\nBrowserslist: caniuse-lite is outdated. Please run:\nnpx browserslist@latest --update-db\n\nWhy you should do it regularly:\nhttps://github.com/browserslist/browserslist#browsers-data-updating\nThe build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process.\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! metalk8s-platform-ui@0.1.0 build: `react-app-rewired --max_old_space_size=3072 build`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the metalk8s-platform-ui@0.1.0 build script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/node/.npm/_logs/2021-03-30T10_19_58_581Z-debug.log\n':
b'\n> metalk8s-platform-ui@0.1.0 build /home/node\n> react-app-rewired --max_old_space_size=3072 build\n\nCreating an optimized production build...\nBrowserslist: caniuse-lite is outdated. Please run:\nnpx browserslist@latest --update-db\n\nWhy you should do it regularly:\nhttps://github.com/browserslist/browserslist#browsers-data-updating\nThe build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process.\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! metalk8s-platform-ui@0.1.0 build: `react-app-rewired --max_old_space_size=3072 build`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the metalk8s-platform-ui@0.1.0 build script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/node/.npm/_logs/2021-03-30T10_19_58_581Z-debug.log\n'
```

--- a/eve/workers/pod-builder/pod.yaml
+++ b/eve/workers/pod-builder/pod.yaml
@@ -32,10 +32,10 @@ spec:
       resources:
         requests:
           cpu: "3"
-          memory: 4Gi
+          memory: 6Gi
         limits:
           cpu: "3"
-          memory: 4Gi
+          memory: 6Gi
       env:
         - name: DOCKER_TLS_CERTDIR
           value: ''

